web/static/js/components/questionnaires/QuestionnaireTitle.jsx

```diff
@@ -11,7 +11,8 @@ class QuestionnaireTitle extends Component {
     t: PropTypes.func,
     dispatch: PropTypes.func.isRequired,
     questionnaire: PropTypes.object,
-    readOnly: PropTypes.bool
+    readOnly: PropTypes.bool,
+    noMenu: PropTypes.bool
```

Comment:
It's better to avoid negative properties. Either:

* rename it to `hideMenu`/`skipMenu`.
* use `menu` with default true value. using `menu={false}`